src/utils/problem.jl

```diff
@@ -1,15 +1,51 @@
+
+"""
+    PolytopeComplement
+
+The complement to a given set. Note that in general, a `PolytopeComplement` is not necessarily a convex set.
+Also note that `PolytopeComplement`s are open by definition.
+
+### Examples
+```julia
+julia> H = Hyperrectangle([0,0], [1,1])
+Hyperrectangle{Int64}([0, 0], [1, 1])
+
+julia> PC = complement(H)
+PolytopeComplement of:
+  Hyperrectangle{Int64}([0, 0], [1, 1])
+
+julia> center(H) ∈ PC
+false
+
+julia> high(H).+[1,1] ∈ PC
+true
+```
+"""
+struct PolytopeComplement{S<:LazySet}
+    P::S
+end
+
+Base.show(io::IO, PC::PolytopeComplement) = (println(io, "PolytopeComplement of:"), println(io, "  ", PC.P))
+LazySets.issubset(s, PC::PolytopeComplement) = LazySets.is_intersection_empty(s, PC.P)
+LazySets.issubset(PC::PolytopeComplement, s) = LazySets.is_intersection_empty(s, PC.P)
```

Comment:
Let's remove this line.

Reply:
Done. Thanks @schillic for pointing this out! That method was never called of course, but now it is also gone! Excited to adopt `LazySets`' `Complement` type and have the pros handle it 😉

Reply:
:wink:
The second method you removed was actually fine (`is_intersection_empty` is symmetric in its arguments).

Reply:
Indeed, but they were both unused, and the PC type will be short lived anyway. 
They were added together and removed together...